### PR TITLE
feat: Enhance HSCodePicker with item details input and workflow integration

### PR DESCRIPTION
**Pull request by mushrafmim**
## Summary

The frontend bumps to the new API Endpoints

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Refactoring (no functional changes)
- [ ] Performance improvement
- [ ] Other (please describe):

## Changes Made

- Bumped to new API Endpoints

## Testing

- [x] I have tested this change locally
- [ ] I have added unit tests for new functionality
- [ ] I have tested edge cases
- [ ] All existing tests pass

## Checklist

- [x] My code follows the project's style guidelines
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked that there are no merge conflicts

## Related Issues

Closes #123 
Fixes #123 
Related to #123 

## Screenshots/Demo

(If applicable, add screenshots or GIFs to help explain your changes)

## Additional Notes

Any additional context, concerns, or notes for reviewers.

## Deployment Notes

(If applicable, include any special deployment instructions or environment considerations)

### CODE REVIEW

File: portals/apps/trader-app/src/services/task.ts [hunk 1]
```diff
@@ -12,15 +12,9 @@ export interface TaskFormData {
   formData: Record<string, unknown>
```
Comment (ginaxu1) [inline portals/apps/trader-app/src/services/task.ts]: review this file `task.ts`. using fetch directly for some calls. should use the wrapper functions `apiGet` and `apiPost` that `api.ts` for consistency and error handling.

**Review by ginaxu1 — CHANGES_REQUESTED** [review]
